src/charts/bar.js

```diff
@@ -830,6 +830,21 @@ define(function(require) {
             dispatcher.call('customClick', e, d, d3Selection.mouse(e), [chartWidth, chartHeight]);
         }
 
+        /**
+         * Gets the percentageAxis, sets it to `percentageAxisToMaxRatio` if all data points are 0
+         * @return {number} Calculated percentageAxis
+         * @private
+         */
+        function getPercentageAxis() {
+            const uniqueDataPoints = new Set(data.map(x => x.value))
```

Comment:
Nit: you can use 'getValue' here

Reply:
This did not work - I tried `x.getValue` and `x.getValue()` both, but did not work.

Reply:
It should be
`const uniqueDataPoints = new Set(data.map(getValue))`

Reply:
Ok, thanks. That works out :-)